data/borealis/Preset.toml

```
@@ -0,0 +1,51 @@
+# URL friendly name
+slug = "borealis"
+# Display name
+name = "Borealis"
+# Authors' name
+creator = "Qaz Cetelic"
+# The commit this preset was built against
+# commit = ""
+# Preset description
+description = "Dark green theme inspired by the northern lights"
+
```

You should probably add a theme version:
```suggestion
version = "1.0.0"
```